src/allmydata/test/cli/test_run.py

```diff
@@ -264,7 +264,7 @@ def test_non_numeric_pid(self):
         self.assertThat(runs, Equals([]))
         self.assertThat(result_code, Equals(1))
 
-    good_file_content_re = re.compile(r"\s[0-9]*\s[0-9]*\s", re.M)
+    good_file_content_re = re.compile(r"\s*[0-9]*\s[0-9]*\s*", re.M)
```

Comment:
... is this relevant to the fixes?

Reply:
Not really at all no, but Hypothesis found a failing case `'0\r0'` which the previous regex didn't match. Could put in a different bug+PR if this one ... keeps failing?

Reply:
Want me to split this out..?